### PR TITLE
Fix #501 and #502

### LINKED ISSUE #501: [V20 Wallet] Add /version endpoint
Add /version endpoint to return wallet version info, including version number, HEAD commit id.

### LINKED ISSUE #502: Set default wallet's label as "Your Wallet"
No wallet name will be shown in GUI if it's the first time to run wallet, cause the default wallet label is empty.

### PR DESCRIPTION
**Pull request by iketheadore**
* Fix #501 , add /version endpoint, which will return the current node version number and the HEAD commit id when build the node
* Fix #502 , set default wallet's label as "Your Wallet"

### CODE REVIEW

File: cmd/skycoin/skycoin.go [hunk 1]
```diff
@@ -29,7 +29,10 @@ import (
 
 var (
 	// Version node version which will be set when build wallet by LDFLAGS
-	Version    = "0.0.0"
+	Version = "0.20.0"
```
Comment (gz-c) [inline cmd/skycoin/skycoin.go]: maybe can add "-dev" to this version

File: src/util/file/file.go [hunk 2]
```diff
@@ -35,6 +35,12 @@ func InitDataDir(dir string) (string, error) {
 		return "", err
 	}
 
+	// check if dir already exist
+	if _, err := os.Stat(dir); !os.IsNotExist(err) {
```
Comment (gz-c) [inline src/util/file/file.go]: this also needs to check the `stat.IsDir()`

File: src/wallet/addresses.go [hunk 3]
```diff
@@ -59,7 +59,7 @@ func GetSkycoinWalletEntry(pub cipher.PubKey, sec cipher.SecKey) ReadableEntry {
 	}
 }
 
-// Returns a ReadableEntry in Bitcoin format
+// GetBitcoinWalletEntry returns a ReadableEntry in Bitcoin format
```
Comment (gz-c) [inline src/wallet/addresses.go]: Does Golint require the function name repeated in the comment string?

File: cmd/skycoin/skycoin.go (hunk 1 above)
Comment (iketheadore) [reply cmd/skycoin/skycoin.go]: yes, I was thinking add suffix as well.

File: src/wallet/addresses.go (hunk 3 above)
Comment (iketheadore) [reply src/wallet/addresses.go]: Yes, every exported function's comment should start with the function name

File: src/util/file/file.go (hunk 2 above)
Comment (iketheadore) [reply src/util/file/file.go]: ok, I will add the check

**Pull request merged**